docs/content/docs/test/Runner.md

```diff
@@ -293,14 +576,48 @@ configurationsContainer.register("Local") {
             }
         }
     }
+}
+```
+
+{{< /tab >}}
+{{< tab "Groovy" >}}
+
+```groovy
+instrumentation {
+    configurations {
+        local {
+            tryToReRunOnTargetBranch = false
+            targets {
+                api27 {
+                    deviceName = "API27"
+                    scheduling {
+                        quota {
+                            retryCount = 1
+                            minimumSuccessCount = 1
+                        }
+
+                        testsCountBasedReservation {
+                            device = new LocalEmulator("27", 27, "Android_SDK_built_for_x86")
```

Comment:
Can we use a factory method here?

Reply:
I think after infra release we can. But I need to change build.gradle in Avito and then documentation.
Consider it's not a blocker.